src/rgw/rgw_sync_module_pubsub.cc

```diff
@@ -663,7 +663,7 @@ class PSSubscription {
             if (old_rule.get_prefix().empty() && 
                 old_rule.get_expiration().get_days() == retention_days &&
                 old_rule.is_enabled()) {
-              ldout(sync_env->cct, 20) << "no need to set lifecycle rule on bucket, existing rule matches config" << dendl;
+              ldpp_dout(sync_env->dpp, 20) << "no need to set lifecycle rule on bucketi, existing rule matches config" << dendl;
```

Comment:
typo here. did you intend to include rgw_sync_module_pubsub.cc in this?

Reply:
I fixed the typo. There were a handful of CR's that are created here (ex: `RGWUserCreateCR, RGWObjectSimplePutCR') and I refactored the logging around them. Looking back I could probably do so for the whole file, but I'm guessing that doesn't have the highest priority right now.